netutils/lib_mapper.py

```diff
@@ -656,6 +660,8 @@
 # Deep copy the reverse, where there is no actual translation happening with special
 # consideration for OS's not in netmiko.
 _MAIN_LIB_MAPPER = copy.deepcopy(NETMIKO_LIB_MAPPER)
+_MAIN_LIB_MAPPER["adva_fsp150f2"] = "adva_fsp150f2"
```

Comment:
These should be added to netmiko, as they are in there: https://ktbyers.github.io/netmiko/PLATFORMS.html